src-tauri/src/commands/accounts.rs

```diff
@@ -41,7 +43,33 @@ pub async fn create_identity(
     account
         .set_active(wn.clone(), &app_handle)
         .await
-        .map_err(|e| format!("Error setting active account: {}", e))
+        .map_err(|e| format!("Error setting active account: {}", e))?;
+
+    // Auto-onboard the account
+    // Publish an inbox relay list
+    publish_relay_list(vec!["wss://auth.nostr1.com".to_string()], 10050, wn.clone()).await?;
+
+    // Publish a key package relay list
+    publish_relay_list(
+        vec![
+            "wss://relay.damus.io".to_string(),
+            "wss://relay.primal.net".to_string(),
+            "wss://nos.lol".to_string(),
+        ],
+        10051,
+        wn.clone(),
+    )
+    .await?;
```

Comment:
Maybe instead of having these relays hard-coded we can read them off a file? That way we can later on modify the default relays without needing to make changes in the code.